systers_portal/templates/account/email.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load account %}
+
+{% block title %} - Email{% endblock %}
+
+{% load crispy_forms_tags %}
+{% block content %}
+
+<center><h1>E-mail Addresses</h1>
+{% if user.emailaddress_set.all %}
+  <p>The following e-mail addresses are associated with your account:</p>
+    <form action="{% url 'account_email' %}" class="email_list" method="post">
+    {% csrf_token %}
+    <fieldset class="blockLabels">
+    {% for emailaddress in user.emailaddress_set.all %}
+       <div class="ctrlHolder">
+         <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
+         <input id="email_radio_{{forloop.counter}}" type="radio" name="email" {% if emailaddress.primary %}checked="checked"{%endif %} value="{{emailaddress.email}}"/>
+         {{ emailaddress.email }}
+         {% if emailaddress.verified %}
+            <span class="verified">Verified</span>
+         {% else %}
+            <span class="unverified">Unverified</span>
+         {% endif %}
+         {% if emailaddress.primary %}<span class="primary">Primary</span>{% endif %}
+         </label>
+       </div>
+    {% endfor %}
+       <div class="buttonHolder">
+          <button class="btn btn-primary" type="submit" name="action_primary" >Make Primary</button>
+          <button class="btn btn-primary" type="submit" name="action_send" >Re-send Verification</button>
+          <button class="btn btn-primary " type="submit" name="action_remove" >Remove</button>
+       </div>
+    </fieldset>
+    </form>
+{% else %}
+    <p><strong>Warning:</strong>You currently do not have any e-mail address set up. You should really add an e-mail address so you can receive notifications, reset your password, etc.</p>
+{% endif %}
+     <h2>Add E-mail Address</h2>
+     <form method="post" action="{% url 'account_email' %}" class="login form-signin">
+          {% csrf_token %}
+          {{ form|crispy }}
+          {% if redirect_field_value %}
+               <input type="hidden" name="{{ redirect_field_name }}"
+                value="{{ redirect_field_value }}"/>
+          {% endif %}
+      <button class="primaryAction btn btn-lg btn-primary btn-block" name="action_add" type="submit">Add E-mail</button>
+     </form></center>
+
+{% endblock %}
+
+{% block extra_body %}
+
+{% endblock %}
```

Comment:
Add a newline to the end of the file and fix indentation: 2 spaces for HTML and CSS :)